scipy/stats/tests/test_stats.py

```diff
@@ -50,6 +50,8 @@
 #  good target for 1.0 status
 X = array([1,2,3,4,5,6,7,8,9], float)
 ZERO = array([0,0,0,0,0,0,0,0,0], float)
+X_UNFLATTENED = array([[1], [2], [3], [4], [5], [6], [7], [8], [9]], float)
```

Comment:
This is better written as `X[:, np.newaxis]`.  And `X_NESTED` equals `X_UNFLATTENED.T`. 

Furthermore, it's best not to add this here, but as normal variables within `test_tvar`.

Reply:
And more importantly, the tests actually don't work as intended. Rather than using an array of shape `(9, 1)`, use something like shape `(9, 7)` and test that for `axis=None` you get a scalar out, and for `axis=0` or `axis=1` you get a result of size 7 or 9.

Reply:
@rgommers I see, thanks for the feedback! I'll remove my `(9, 1)` test and add the `(9, 7)` test into `test_tvar`.